graphs/minimum_spanning_tree_kruskal.py

```diff
@@ -1,15 +1,14 @@
-from typing import List, Tuple
-
-
-def kruskal(num_nodes: int, num_edges: int, edges: List[Tuple[int, int, int]]) -> int:
```

Comment:
Thankyou for removing the redundant variables, but the type hints (which are imported from `typing`) are essential

Reply:
`List`, `Tuple` are deprecated as type-hint since [python 3.9](https://docs.python.org/3/library/typing.html#typing.Tuple), Instead It supports built-in `list`, tuple`.

Reply:
Thanks for pointing it out, I was not aware of this 👍